src/main/java/com/meraphorce/services/UserService.java

```diff
@@ -1,23 +1,57 @@
 package com.meraphorce.services;
 
+import com.meraphorce.dto.UserResponse;
+import com.meraphorce.dto.UserRequest;
 import com.meraphorce.models.User;
 import com.meraphorce.respositories.UserRepository;
+import com.meraphorce.mappers.impl.UserMapper;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 import java.util.List;
+import java.util.stream.Collectors;
 
 @Service
-public class UserService {
-
+public class UserService
+{
     @Autowired
     private UserRepository userRepository;
 
-    public User createUser(User user){
-        return userRepository.save(user);
+    @Autowired
+    private UserMapper mapper;
+
+    public UserResponse createUser(UserRequest user) throws UserAlreadyExistsException {
```

Comment:
There is not need to throw the UserAlreadyExistsException, you can remove that line as it is a Non Checked exception and does not require to be handled